content/techniques/caching.md

```diff
@@ -161,6 +161,42 @@ import { AppController } from './app.controller';
 export class ApplicationModule {}
 ```
 
+##### Redis cache `set` and `get` example
+
```

Comment:
I'm sorry, I don't understand what this example is trying to do, or how the code is working, without some greater context.  Could you please try to explain:
1. What is the goal of this section
2. What are the relevant portions of the API shown in the example and any concepts needed to understand them (i.e., this is the first time we're seeing `CacheManager`, so we need some context to understand how it works).

Thank you.

Reply:
This section adds simple example how to use cache (e.g. Redis cache) in your code. The current documention is nice but does not include any example or docs about how to simply set "values" by "keys" in Redis cache and get it. 

Like for e.g 

[Yii framework's docs](https://www.yiiframework.com/doc/guide/2.0/en/caching-data) 

```php
// try retrieving $data from cache
$data = $cache->get($key);

if ($data === false) {
    // $data is not found in cache, calculate it from scratch
    $data = $this->calculateSomething();

    // store $data in cache so that it can be retrieved next time
    $cache->set($key, $data);
}

// $data is available here
```

[Laravel's Docs](https://laravel.com/docs/7.x/cache)

```php
Cache::put('key', 'value', $seconds);

$value = Cache::get('key');
```

[`CacheManager`](https://github.com/BryanDonovan/node-cache-manager) provide generic API regardless of what is used to store cache e.g. Redis, MySQL, file  etc.

Feel free to let me know if more info needed